lib/reports/project_translation_report.rb

```diff
@@ -28,20 +28,18 @@ def self.generate_csv(start_date, end_date)
                                    .group(['projects.name','translations.rfc5646_locale'])
                                    .sum(:words_count)
         languages = translations.keys.map {|k| k[1]}.uniq.sort
-        empty_cols = Array.new(languages.count, '')
+        empty_cols = Array.new(languages.count - 1, '')
 
         csv << ['Start Date', start_date] + empty_cols
         csv << ['End Date', end_date] + empty_cols
-        csv << ['', ''] + empty_cols
-        csv << ['Project', 'source'] + languages
+        csv << [''] + empty_cols
```

Comment:
This csv line will be one column short, as `empty_cols` already decreased one column

Reply:
Oh, thanks for pointing that out

Reply:
I’m not sure why it was confusing.

Reply:
The source column was confusing because it was the total, and not actually the distinct source words.  When we talked about it, we decided that a total column wasn't particularly useful, so we'd just get rid of this, but if you think the total column is useful, we could rename it instead.

Reply:
I will chat with Ace and get back to you to rename or to remove.